src/Fixer/Casing/NativeFunctionCasingFixer.php

```diff
@@ -67,6 +68,9 @@ protected function applyFix(\SplFileInfo $file, Tokens $tokens)
             }
 
             $functionNamePrefix = $tokens->getPrevMeaningfulToken($index);
+            if ($tokens[$functionNamePrefix]->isGivenKind(CT::T_RETURN_REF)) {
+                $functionNamePrefix = $tokens->getPrevMeaningfulToken($functionNamePrefix);
+            }
             if ($tokens[$functionNamePrefix]->isGivenKind([T_DOUBLE_COLON, T_NEW, T_OBJECT_OPERATOR, T_FUNCTION])) {
```

Comment:
I would add `CT::T_RETURN_REF` to this array instead.

Reply:
Thanks, I thought `CT::T_RETURN_REF` would also be in `$next1 = & Next($array1);` case, but it is not.